test/tests/functional/pbs_job_array.py

```diff
@@ -744,3 +782,203 @@ def test_recover_big_array_job(self):
             self.fail('TC failed as server recovery failed')
         else:
             self.server.expect(JOB, {ATTR_state: 'B'}, id=j_id)
+
+    @skipOnCpuSet
+    def test_max_run_subjobs_basic(self):
+        """
+        Test that if a job is submitted with 'max_run_subjobs' attribute
+        number of subjobs that run do not exceed the attribute value.
+        """
+
+        a = {'resources_available.ncpus': 8}
+        self.server.manager(MGR_CMD_SET, NODE, a, self.mom.shortname)
+        j = Job(attrs={ATTR_J: '1-20%2'})
+        j_id = self.server.submit(j)
+        self.server.expect(JOB, {ATTR_state: 'B'}, id=j_id)
+        self.server.expect(JOB, {'job_state=R': 2}, extend='t')
+
+        self.server.alterjob(j_id, {ATTR_W: 'max_run_subjobs=5'})
+        a = {'scheduling': 'True'}
+        self.server.manager(MGR_CMD_SET, SERVER, a)
```

Comment:
This manager() call can fit within 79 characters.  No need to split it into 2 lines.